packages/tailwindcss/src/candidate.test.ts

```diff
@@ -1228,7 +1204,7 @@ it('should not parse compound group with a non-compoundable variant', () => {
   utilities.static('flex', () => [])
 
   let variants = new Variants()
-  variants.compound('group', () => {})
+  variants.compoundWith('group', ['selector'], () => {})
```

Comment:
This seems wrong I think
```suggestion
  variants.compoundWith('group', Compounds.StyleRules, () => {})
```